lib/atlas/scaler/time_curve_scaler.rb

```diff
@@ -0,0 +1,29 @@
+module Atlas
+  class Scaler::TimeCurveScaler
```

Comment:
It would be nice to have some documentation for this class. The method signatures of `call` and `initialize` don't give much clue as to what the parameters should be, or what the result values are (if any).

For example: are `base_dataset` and `derived_dataset` keys for datasets (no), or actual dataset objects (yes). Is it acceptable for both to be any `Dataset` subclass, or does `base` have to be a `Full` and `derived` a `Derived`?

Does `scale` return the scaled CSV file when it has completed, or is the return value nothing useable?